HamiltonB/main.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+int n=0;
+int main()
+{
+    string *people=NULL;
+    string message,line;
+    ifstream file;
+    file.open ("UsersData.txt");
+    if (file.is_open())
+        while(getline(file,line))
+            ++n;
+        people = new string[n];
+        file.clear();
+        file.seekg(0);
+        for(int i=0;getline(file,line);++i)
+            people[i] = line;
+        cin >> message;
+        if (message == "display")
+            for(int i=0;i<n;++i)
+                cout << people[i] << '\n';
```

Comment:
I think it should print possible options with `1`, `2`, `3`... instead of forcing the user to type `display`.